lib/gds_api/config.rb

```diff
@@ -0,0 +1,17 @@
+module GdsApi
+  def self.configure
+    yield(config)
+  end
+
+  def self.config
```

Comment:
Generally yeah, because it's more obvious what's going on when inputs are explicitly declared. If it can be changed anywhere in the app then you have to look in multiple places to know how the client will behave.

Not a massive issue if it's temporary code though.